### ISSUE TITLE
By section name is bad.

### ISSUE THREAD

**Issue by TanakaYasen** [changeofpace/Self-Remapping-Code#2]
As the title is, do you consider that: 
```
#define SCNVIEW_MASK_READ	0x1
#define SCNVIEW_MASK_WRITE	0x2
#define SCNVIEW_MASK_EXECUTE	0x4 
		if (pSectionHeader->Characteristics & IMAGE_SCN_MEM_READ) mask |= SCNVIEW_MASK_READ;
		if (pSectionHeader->Characteristics & IMAGE_SCN_MEM_WRITE) mask |= SCNVIEW_MASK_WRITE;
		if (pSectionHeader->Characteristics & IMAGE_SCN_MEM_EXECUTE) mask |= SCNVIEW_MASK_EXECUTE; 
// then map `mask` to `PAGE_*`
```

**Comment by changeofpace** [comment]
Hello, sorry for the late response.

Are you suggesting that the remap logic should use the section page protection values defined in the pe header instead of using hard-coded protection values based on section name?

**Comment by changeofpace** [comment]
Good idea. I'll fix this when I get around to updating this project.

Thanks for the suggestion.

**Comment by changeofpace** [comment]
Fixed in 886c6a6a9c17cd18f7d72eb51f3ef3abdfac7d92
